pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -971,7 +971,7 @@ func (og *operationGenerator) GenerateUnmountDeviceFunc(
 			return volumetypes.NewOperationContext(eventErr, detailedErr, migrated)
 		}
 
-		klog.Infof(deviceToDetach.GenerateMsg("UnmountDevice succeeded", ""))
+		klog.Infof(deviceToDetach.GenerateMsgDetailed("UnmountDevice succeeded", ""))
```

Comment:
While at it, please replace with `klog.Info`. `Infof` interprets the first parameter as a format string, which wasn't (and still isn't) what gets passed here.

Reply:
@swiftslee: please address this comment.

Reply:
> @swiftslee: please address this comment.

sorry for the miss. done